akka-docs/src/main/paradox/.htaccess

```diff
@@ -24,4 +24,4 @@ RedirectMatch 301 ^(.*)/stream/operators/Source-or-Flow/UnzipWith\.html$ $1/stre
 
 RedirectMatch 301 ^(.*)/stream/operators/Source-or-Flow/apply\.html$ $1/stream/operators/Source/from.html
 
-RedirectMatch 301 ^(.*)/guide/(.*) $1/typed/guide/$2
\ No newline at end of file
+RedirectMatch 301 ^(.*[^d])/guide/(.*) $1/typed/guide/$2
```

Comment:
I think this could be a bit fragile. It won't match if the URL prefix contains a "d"? 

Using a negative look-behind seems to have the right behavior (at least in JavaScript) and is maybe a bit more clear:
`^(.*)(?<!/typed)/guide\/(.*)`

```javascript
r1 = /^(.*)(?<!\/typed)\/guide\/(.*)/
// => /^(.*)(?<!\/typed)\/guide\/(.*)/
r1.exec("http://doc.akka.io/docs/akka/snapshot/typed/guide/introduction.html")
// => null
r1.exec("https://doc.akka.io/docs/akka/snapshot/guide/introduction.html")
// => (3) ["https://doc.akka.io/docs/akka/snapshot/guide/introduction.html", "https://doc.akka.io/docs/akka/snapshot", "introduction.html", index: 0, input: "https://doc.akka.io/docs/akka/snapshot/guide/introduction.html", groups: undefined]
```

I think that's supported by Apache `mod_alias`, which uses PCRE for regex.

Reply:
I was too slow :)

Reply:
sorry ;)

> It won't match if the URL prefix contains a "d"?

It won't match if the URL prefix ends with a "d". Not super elegant, but we expect the prefix to be either 'current', 'snapshot' or a version number, so that doesn't seem too bad (though it would fail if we ever decided to publish snapshots with versions ending in their git hash). I'm not sure it's worth adding the complexity of a negative look-behind.

Reply:
Well you wouldn't have to implement the matcher :) but not worth changing now that it's merged, I agree.